### PR TITLE
Add optional parameters to Process Account task

### PR DESCRIPTION
**Pull request by cruffinoni**
The task to ingest account data can now take 2 optional parameters. It allow to ingest data at a specific month & year like master spreadsheet task does.

### CODE REVIEW

**Comment by cruffinoni** [comment]
Quick explanation of the bug solved on the last commit:
When performing the task of acquiring billing data and creating a file identifier, if the current file already has the same type (ec2,...), the same time stamp as an existing file but not the same type of report (daily vs. monthly report); the task generates the same file identifier and refuses consumption. We have changed the way identities are created to solve the problem.

File: aws/usageReports/instanceCount/utils.go [hunk 1]
```diff
@@ -85,11 +85,13 @@ func generateId(report InstanceCountReport) (string, error) {
 		ReportDate time.Time `json:"reportDate"`
 		Type       string    `json:"type"`
 		Region     string    `json:"region"`
+		ReportType       string    `json:"reportType"`
```
Comment (hug33k) [inline aws/usageReports/instanceCount/utils.go]: Indentation

File: server/taskProcessAccount.go [hunk 2]
```diff
@@ -19,6 +19,9 @@ import (
 	"database/sql"
 	"errors"
 	"flag"
+	"github.com/trackit/trackit/aws/usageReports/riEc2"
+	"github.com/trackit/trackit/aws/usageReports/riRdS"
+	onDemandToRiEc2 "github.com/trackit/trackit/onDemandToRI/ec2"
```
Comment (hug33k) [inline server/taskProcessAccount.go]: Wrong import order

**Review by hug33k — CHANGES_REQUESTED** [review]

File: aws/usageReports/history/history.go [hunk 3]
```diff
@@ -222,14 +222,21 @@ func CheckBillingDataCompleted(ctx context.Context, startDate time.Time, endDate
...
+	} else {
+		startDate = date
+		endDate = time.Date(date.Year(), date.Month()+1, 0, 23, 59, 59, 999999999, date.Location())
+	}
 	logger.Info("Starting history report", map[string]interface{}{
 		"awsAccountId": aa.Id,
 		"startDate":    startDate.Format("2006-01-02T15:04:05Z"),
 		"endDate":      endDate.Format("2006-01-02T15:04:05Z"),
 	})
+	fmt.Printf("Start history at '%v' & '%v'\n", startDate, endDate)
```
Comment (hug33k) [inline aws/usageReports/history/history.go]: Please remove

**Review by hug33k — CHANGES_REQUESTED** [review]

**Review by jgchoppe — APPROVED** [review]

**Review by hug33k — APPROVED** [review]

**Pull request merged**